sphinx_needs/directives/needflow.py

```diff
@@ -489,11 +489,22 @@ def process_needflow(app: Sphinx, doctree: nodes.document, fromdocname: str, fou
             para += filter_node
             content.append(para)
 
+        # If the central PlantUML Debug option is set, we need to generate the code already once in advance and
+        # report any errors directly.
+        if app.config.needs_plantuml_debug:
```

Comment:
Perhaps it would be better to run this in a transform, that goes through and finds all (sphinx-needs generated) `plantuml` nodes,
rather than having to add it to every different directive